rust/ql/lib/codeql/rust/internal/Type.qll

```diff
@@ -7,6 +7,15 @@ private import codeql.rust.internal.CachedStages
 private import codeql.rust.elements.internal.generated.Raw
 private import codeql.rust.elements.internal.generated.Synth
 
+/** Holds if a dyn trait type should have a type parameter associated with `n`. */
```

Comment:
An example would be good here.

Also, I think this predicate could / should be private?